boards/pic32-wifire/Makefile.include

```diff
@@ -3,3 +3,22 @@ export CPU_MODEL=p32mz2048efg100
 export INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include/
 export APPDEPS += $(RIOTCPU)/$(CPU)/$(CPU_MODEL)/$(CPU_MODEL).S
 export USE_UHI_SYSCALLS = 1
+
+PORT_LINUX ?= /dev/ttyUSB0
+BAUD ?= 9600
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# running the terminal does reset the board
+RESET ?= echo
+RESET_FLAGS ?= "No RESET but opening UART does a reset"
+
+# pic32prog
+#
+# For PICkit3:
+#
+# * Connect the chipKIT-Wi-Fire to USB
+# * Connect the PICkit3 to ICSP holes
+# * https://docs.creatordev.io/wifire/guides/wifire-programming/
+# * The Arrow `▶` pin goes into the pin number 1 (with a square shape around it)
```

Comment:
Wasn't there a discussion already about unicode characters? I don't remember the outcome

Reply:
I wanted to be fancy on this one :D I can replace by `|>` and even say "triangle" because it is more a triangle than an arrow.